### DISCUSSION TITLE
Remove Navigation of Resource

### DISCUSSION THREAD

**Discussion by Timo-Reist** [filamentphp/filament#13762] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

<img width="380" alt="image" src="https://github.com/user-attachments/assets/a61971e1-a761-47c0-96f8-6bae1df8fa85">

How can I hide / remove the navigation of a resource on the create/edit/list pages? I can't find anything about it, maybe Im overlooking something. Thank you for your help.

**Comment by sinnbeck** [comment, marked as answer]
They are called breadcrumbs. Just add the following method to each page
```php
  public function getBreadcrumbs(): array
    {
        return [];
    }

**Reply by Timo-Reist** [reply]
Thank you :)
